shell.core/src/main/java/org/metaborg/spoofax/shell/client/Repl.java

```diff
@@ -47,6 +37,7 @@
         this.display = display;
         this.invoker = invoker;
 
+        // TODO: this does not really belong here.
```

Comment:
The reason i moved these here is that in my opinion the prompt should look the same across different implementations for viewers. It is then up to the viewer to decide in what way to convey this look to the user.

I'm afraid that setting different defaults in each implementation could confuse the user as to what shell they're using.

That being said, I also think there's nothing wrong with exposing hooks to shell.console / shell.eclipse that allow the user to change the look of the prompt :p